isis/CMakeLists.txt

```diff
@@ -513,9 +513,11 @@ add_dependencies(isis3 appdata)
 # Add custom build target to copy modified template files to the build/appdata/templates
 # directory. (As above - may need to be updated)
 add_custom_target(templates ALL COMMAND ${CMAKE_COMMAND} -E copy_if_different
-  ${CMAKE_SOURCE_DIR}/appdata/templates/*.pvl ${CMAKE_BINARY_DIR}/appdata/templates)
+  ${CMAKE_SOURCE_DIR}/src/*/templates/*.pvl ${CMAKE_BINARY_DIR}/appdata/templates/apollo/)
```

Comment:
following what we just talked about, this should be changed to *.def instead of *.pvl (at least when the general solution is added, I can work with this for now)